### PR TITLE
Uyuni 2024.08

### PR DESCRIPTION
**Pull request by raulillo82**
Documentation will be added as soon as ready.

- [x] doc
- [x] final release notes (unless it is not released on the expected date)
- [ ] final release date (shifted from 22nd to 26th, unless something happens, which means readapting everything)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -233,22 +233,22 @@ <h3>Latest news</h3>
...
           <!-- News box Start -->
-          <div class="news-box">
+	  <div class="news-box">
             <div class="news-box-title">          
-              <span>19/07/2024</span>
+              <span>22/08/2024</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2024.07 is now available with new products enabled, changed behaviour of repo-sync, Confidential Computing Attestation UI and more! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.08 is now available with configuration of AppStreams via Activation Keys, migration "prepare" command, mgrprxy support config and more! This is the last version to support the RPM based product before moving to a containers only release! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
Comment (juliogonzalez) [inline index.html:241]:
```suggestion
              <p>Uyuni 2024.08 is now available with configuration of AppStreams via Activation Keys, migration "prepare" command, mgrprxy support config and more! This is the last version to support the RPM based product before moving to a containers only release! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details and for a very important known issue!</p>
```
And maybe adjust the same at `news.html`

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (raulillo82) [reply index.html]: Fixed in both places.

**Comment by raulillo82** [comment]
PR adjusted with the new release date (and the new release notes with that date). Docs still uncomplete.

**Comment by raulillo82** [comment]
Documentation ready (pdf's added into this PR), release notes were already refreshed this morning.
In my opinion this PR is ready to merge as soon as release is done (on Monday).

**Pull request merged**